src/test/java/org/opentripplanner/routing/algorithm/TestBanning.java

```diff
@@ -1,203 +1,73 @@
 package org.opentripplanner.routing.algorithm;
 
-import junit.framework.TestCase;
-import org.junit.Ignore;
-import org.opentripplanner.ConstantsForTests;
-import org.opentripplanner.common.model.T2;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opentripplanner.model.Agency;
 import org.opentripplanner.model.FeedScopedId;
-import org.opentripplanner.routing.algorithm.astar.AStar;
+import org.opentripplanner.model.Route;
 import org.opentripplanner.routing.core.RoutingRequest;
-import org.opentripplanner.routing.core.State;
-import org.opentripplanner.routing.graph.Graph;
-import org.opentripplanner.routing.graph.Vertex;
-import org.opentripplanner.routing.request.BannedStopSet;
-import org.opentripplanner.routing.spt.GraphPath;
-import org.opentripplanner.routing.spt.ShortestPathTree;
-import org.opentripplanner.util.TestUtils;
-
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Random;
-import java.util.Set;
+
+import java.util.Arrays;
+import java.util.Collection;
 
 /**
- * TODO OTP2 - Test is too close to the implementation and will need to be reimplemented.
+ * Test the banning and whitelisting functionality in the RoutingRequest
```

Comment:
Expand Javadoc to state that this is only testing the expansion of whitelist and banning lists in the request, but not actually testing the routing (which could be added later). Or: make it clear which integration / full stack tests exist that are hitting these code paths.